apps/nova/src/components/layout/structure.tsx

```
@@ -36,85 +73,37 @@ export default function Structure({
+      defaultLayout={defaultLayout}
+      navCollapsedSize={navCollapsedSize}
+      isCollapsed={isCollapsed}
+      setIsCollapsed={setIsCollapsed}
+      debouncedSaveSizes={debouncedSaveSizes}
+      debouncedSaveCollapsed={debouncedSaveCollapsed}
+      mobileHeader={mobileHeader}
+      sidebarHeader={sidebarHeader}
+      sidebarContent={sidebarContent}
+      actions={actions}
+      userPopover={userPopover}
+      children={children}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)